src/azul/strings.py

```diff
@@ -48,12 +48,15 @@ def pluralize(word: str, count: int) -> str:
     return result
 
 
-def unwrap(string: str):
+def unwrap(string: str, keep_paragraphs: bool = False) -> str:
```

Comment:
I think unwrap should only ever `keep_paragraphs` and it shouldn't be an option. If you're worried about this breaking something, double check the usages of `unwrap` to be sure (but I don't think it will). 

Also I recall @hannes-ucsc saying that the body of `unwrap` could just be replaced with a call to `dedent`. If this is done, the name "unwrap" doesn't really make sense and perhaps should be replaced with `format_description` or something? IDK. I think @hannes-ucsc might have said something about it during our call on Friday, but I can't remember.

Reply:
Yes, I think `unwrap` should always call `dedent` and not be modal. The name `unwrap` doesn't make sense. I'll leave it to you if you want to rename it to `format_description` as @jessebrennan  suggests or if you'd rather create a ticket for the rename.